src/templates/DetailsPage.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useEffect, useState } from "react";
+import { PageProps } from "gatsby";
+import { DetailsPageScreen } from "@forkfacts/screens";
+import { SEO } from "@forkfacts/components";
+
+import { generateRdiForFood, getAgeRangesForLifeStage, getValueRounded } from "@forkfacts/helpers";
+import { Box } from "@mui/material";
+import { lifeStageItems, menuItems, tabItems } from "../RealData/realData";
+import { useStore } from "../store/store";
+import {
+  NutrientGroup,
+  NutrientItem,
+  NutritionFact,
+  NutritionTableRow,
+  SelectedNutrient,
+} from "@forkfacts/models";
+
+const DynamicPageTemplate = ({ pageContext }: PageProps) => {
+  const { food, recommendedDailyIntakes } = pageContext as any;
+  const [rows, setRows] = useState<any[]>([]);
+  const { selectedLifeStage, selectedAge, selectedNutrients, setSelectedAge } = useStore(
+    (state) => state
+  );
+  const thisFood = food as any;
+  const allRdis = recommendedDailyIntakes as any[];
+  const nutritionFacts: NutritionFact[] = generateRdiForFood(thisFood, allRdis);
+
+  const getSelectedNutrients = (nutrients: SelectedNutrient[]) => {
```

Comment:
This functions seems duplicate of `getSelectedNutrients` in `utils.ts`. Please remove one of them

Reply:
No I am not sure of that. I just confirmed that